.github/workflows/test.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
       matrix:
         os: [windows-latest, ubuntu-16.04, ubuntu-18.04, ubuntu-20.04]
```

Comment:
There are some issues (in the downloading data phase) on MacOS when running with other tests. So we split it into an individual config.